distribution/ddf-common/src/main/resources/etc/log4j2.config.xml

```diff
@@ -12,7 +12,7 @@
  *
  **/
 -->
-<Configuration status="WARN">
+<Configuration status="FATAL">
```

Comment:
Change this to FATAL to suppress some noise in the logs, but hopefully still show useful log messages in case logging fails.